app/controllers/users_controller.rb

```diff
@@ -64,4 +67,20 @@ def should_display_user?(user)
   def get_user
     @user = User.friendly.find(params[:id])
   end
+
+  def count_timezone_degrees
+    unless params[:timezone_select].blank?
```

Comment:
yes but `params[:timezone_select]` is hold the selected option and filter get the `params[:timezone_filter]` what is different. It's a little bit confusing maybe, but I didn't want to overwrite what selected the user.